package.json

```diff
@@ -37,7 +37,7 @@
     "prop-types": "15.6.0",
     "react": "16.2.0",
     "react-dom": "16.2.0",
-    "react-stars": "^2.2.5",
+    "react-rating": "^1.0.5",
```

Comment:
If you'll change this to 1.0.5 instead of ^1.0.5, it will probably be better. We get upgrading PRs from greenkeeper right away, so being kept behind shouldn't be a problem.